Mathlib/CategoryTheory/Comma/Over/Basic.lean

```diff
@@ -768,6 +768,29 @@ instance isEquivalence_toUnder (X : T) (F : D ⥤ T) [F.IsEquivalence] :
 end StructuredArrow
 
 namespace Functor
+variable {C D : Type*} [Category C] [Category D] {X : C} {F : C ⥤ D}
```

Comment:
Let's use the categories that are already defined near the top of this file, rather than making new ones with new names just for these lemmas